### PR TITLE
feat(connect): add changeWipeCode method

### PR DESCRIPTION
**Pull request by mroz22**
connect implementation of https://trezor.io/learn/a/create-wipe-code-to-erase-device
use with caution. 
cc @hynek-jina

### CODE REVIEW

File: packages/connect/src/api/changeWipeCode.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// origin: https://github.com/trezor/connect/blob/develop/src/js/core/methods/ChangePin.js
+
```
Comment (tomasklim) [inline packages/connect/src/api/changeWipeCode.ts]:
```suggestion
```

**Review by tomasklim — DISMISSED** [review]
Could you please explain what it does?

**Comment by mroz22** [comment]
@tomasklim edited PR description

File: packages/connect/src/api/changeWipeCode.ts (hunk 1 above)
Comment (mroz22) [reply packages/connect/src/api/changeWipeCode.ts]: [084a57a](https://github.com/trezor/trezor-suite/pull/10407/commits/084a57a2bc65936dc323902d5f03406af64bedf1)

**Review by hynek-jina — APPROVED** [review]
LGTM

**Pull request merged**